litestar/openapi/controller.py

```diff
@@ -133,7 +133,12 @@ def render_methods_map(self) -> dict[Literal["redoc", "swagger", "elements"], Ca
             "elements": self.render_stoplight_elements,
         }
 
-    @get(path="/openapi.yaml", media_type=OpenAPIMediaType.OPENAPI_YAML, include_in_schema=False, sync_to_thread=False)
+    @get(
+        path=["/openapi.yaml", "openapi.yml"],
```

Comment:
```suggestion
        path=["/openapi.yaml", "/openapi.yml"],
```
Should it have the prefix here?